src/plone/app/mosaic/utils.py

```diff
@@ -98,7 +99,7 @@ def extractFieldInformation(schema, context, request, prefix):
                 }
 
 
-def getContentLayoutsForType(pt):
+def getContentLayoutsForType(pt, context=None):
```

Comment:
Out of scope here: this should be just an adapter on the context where we want the layout to be looked up for and not just a method. This way we could provide a generic adapter for ``*`` and if one needs it in a project it could be anything.